mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,274 @@
+    <string name="preference_leakcanary">LeakCanary</string>
+    <!-- Preference category for developer tools -->
+    <string name="developer_tools_category">Developer tools</string>
+    <!-- Preference for developers -->
+    <string name="preferences_remote_debugging">Remote debugging via USB</string>
+    <!-- Preference title for switch preference to show search suggestions -->
+    <string name="preferences_show_search_suggestions">Show search suggestions</string>
+    <!-- Preference for account settings -->
+    <string name="preferences_account_settings">Account settings</string>
+
+    <!-- Account Preferences -->
+    <!-- Preference for triggering sync -->
+    <string name="preferences_sync_now">Sync now</string>
+    <!-- Preference category for sync -->
+    <string name="preferences_sync_category">Choose what to sync</string>
+    <!-- Preference for syncing history -->
+    <string name="preferences_sync_history">History</string>
+    <!-- Preference for signing out -->
+    <string name="preferences_sign_out">Sign out</string>
+    <!-- Label indicating that sync is in progress -->
+    <string name="sync_syncing">Syncing</string>
+    <!-- Label summary indicating that sync failed along with last time it succeeded -->
+    <string name="sync_failed_summary">Sync failed. Last success: %s</string>
+    <!-- Label summary showing never synced -->
+    <string name="sync_failed_never_synced_summary">Sync failed. Last synced: never</string>
+    <!-- Label summary showing last time synced -->
+    <string name="sync_last_synced_summary">Last synced: %s</string>
+    <!-- Label summary showing never synced -->
+    <string name="sync_never_synced_summary">Last synced: never</string>
+
+    <!-- Advanced Preferences -->
+    <!-- Preference switch for Telemetry -->
+    <string name="preferences_telemetry">Telemetry</string>
+    <!-- Preference switch for crash reporter -->
+    <string name="preferences_crash_reporter">Crash reporter</string>
+    <!-- Preference switch for Mozilla location service -->
+    <string name="preferences_mozilla_location_service">Mozilla location service</string>
+    <!-- Preference switch for app health report. -->
+    <string name="preferences_fenix_health_report">Fenix health report</string>
+
+    <!-- Quick Action Sheet -->
+    <!-- Option in Quick Action Sheet in the browser to share the current page -->
+    <string name="quick_action_share">Share</string>
+    <!-- Option in Quick Action Sheet in the browser to download the current page -->
+    <string name="quick_action_download">Download</string>
+    <!-- Option in Quick Action Sheet in the browser to bookmark the current page -->
+    <string name="quick_action_bookmark">Bookmark</string>
+    <!-- Option in Quick Action Sheet in the browser to put the the current page in reader mode -->
+    <string name="quick_action_read">Read</string>
+    <!-- Content description (not visible, for screen readers etc.): Quick action sheet-->
```

Comment:
Probably better to explain simply what a Quick action sheet is

Reply:
Quick action sheet aka quick action bar is a sliding panel that contains buttons to bookmark, share, and enter reading mode for MVP and will grow to include buttons for other page-level actions in future releases.

Reply:
ok thanks. That's also something that should be called out in the comments